projects/com.oracle.truffle.llvm.runtime/src/com/oracle/truffle/llvm/runtime/options/LLVMOptionServiceProvider.java

```diff
@@ -29,8 +29,5 @@
  */
 package com.oracle.truffle.llvm.runtime.options;
 
-import java.util.List;
-
 public interface LLVMOptionServiceProvider {
```

Comment:
The information is up-to-date. One still needs to implement LLVMOptionsServiceProvider so that mx su-options works.